src/components/Button.spec.js

```diff
@@ -0,0 +1,34 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import App from './App'
+import Button from './Button'
+
+it('renders a Button', () => {
+  render(<Button />)
+  expect(screen.getByRole('button')).toBeVisible()
+})
+
+it('renders a Button with textContent "Spin!"', () => {
+  render(<App />)
+  expect(screen.getByText('Spin!')).toBeVisible()
+})
```

Comment:
To get the test a bit saver, you could use `expect(
      screen.getByRole('button', { name: 'BUTTONTEXT' })
    ).toBeInTheDocument()` and give your button an aria-label with the "BUTTONTEXT"
    
    `GetByText` will get green only for the text, even when it´s in the header